deploy/helm/values.yaml

```diff
@@ -468,3 +468,19 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+additionalPolicies:
```

Comment:
As far as I know, the Conftest is deprecated and planned for removal. If that's _not_ the case, perhaps it would be good to unify this with `conftest.library` and `conftest.policy`.

Reply:
I'm also not attached to this name, but it seems to follow in the tradition of other charts in the wild.